Dockerfile

```diff
@@ -0,0 +1,12 @@
+# Build controller-manager
+ARG GO_VERSION
+FROM golang:$GO_VERSION AS builder
```

Comment:
maybe need to prefix it with quay.io / docker.io etc (preferable non docker.io because the limits) else it won't work with podman that doesn't have the shortcut

Reply:
no sure I catch you drift here

Reply:
golang:version is a short name
you need to prefix it with either docker.io, quay.io etc
try to run it without podman, it will ask you to select the registry (depends on how it is configured)

Reply:
my local is running it with docker but I don't see the issue you talk about:
```
$ make container
docker build --build-arg GO_VERSION=1.17 . -t quay.io/kubevirt/kubemacpool:latest
Sending build context to Docker daemon  249.3MB
Step 1/8 : ARG GO_VERSION
Step 2/8 : FROM golang:$GO_VERSION AS builder
 ---> 742df529b073
Step 3/8 : WORKDIR $GOPATH/src/github.com/k8snetworkplumbingwg/kubemacpool
 ---> Using cache
 ---> 5a95877f7ab6
Step 4/8 : COPY . .
 ---> 180335fac869
Step 5/8 : RUN GOFLAGS=-mod=vendor GO111MODULE=on CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o /manager $GOPATH/src/github.com/k8snetworkplumbingwg/kubemacpool/cmd/manager
 ---> Running in af4b8ae6258e
Removing intermediate container af4b8ae6258e
 ---> d986ff9e3f89
Step 6/8 : FROM registry.access.redhat.com/ubi8/ubi-minimal
 ---> 4df334db583d
Step 7/8 : COPY --from=builder /manager /
 ---> Using cache
 ---> 86a7c82e9bc4
Step 8/8 : CMD ["/manager"]
 ---> Using cache
 ---> d7a768f2e093
Successfully built d7a768f2e093
Successfully tagged quay.io/kubevirt/kubemacpool:latest
```

Reply:
I meant with podman (not without podman)